smartsim/experiment.py

```diff
@@ -184,6 +193,19 @@ def start(
 
         :type kill_on_interrupt: bool, optional
         """
+
+        # Check that entity started is a database and add the db identifier
+
+        for index, item in enumerate(args):
+            if isinstance(item, (Orchestrator, Model, Ensemble)):
+                self.my_list_after_start.append(args[index].__dict__["db_identifier"])
```

Comment:
`my_list_after_start` -- the name should refer somehow to database identities so that code referencing it is clear